comixed-frontend/src/assets/i18n/pt/library.json

```diff
@@ -169,378 +169,378 @@
+      "header": "Remover os comics selecionados de {name}",
+      "message": "Tem a certeza que quer remover {count, plural, =1{um comic} other{# comics}} desta lista de leitura?"
     }
   },
   "reading-lists-page": {
-    "title": "ComiXed: Reading Lists - {count, plural, =1{One List} other{# Lists}}"
+    "title": "ComiXed: Listas de leitura - {count, plural, =1{Uma lista} other{# listas}}"
   },
   "reading-list-page": {
-    "title": "ComiXed: Reading List - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Lista de leitura - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "publishers-page": {
-    "title": "ComiXed: Publishers List ({count})"
+    "title": "ComiXed: Lista de editoras ({count})"
   },
   "publisher-details-page": {
-    "title": "ComiXed: Publisher - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Editora - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "series-page": {
-    "title": "ComiXed: Series List ({count})"
+    "title": "ComiXed: Lista de series ({count})"
   },
   "series-details-page": {
-    "title": "ComiXed: Series - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Serie - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "characters-page": {
-    "title": "ComiXed: Characters ({count})"
+    "title": "ComiXed: Personagems ({count})"
   },
   "character-details-page": {
-    "title": "ComiXed: Character - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Personagem - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "teams-page": {
-    "title": "ComiXed: Teams ({count})"
+    "title": "ComiXed: Equipas ({count})"
   },
   "team-details-page": {
-    "title": "ComiXed: Team - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Equipa - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "locations-page": {
-    "title": "ComiXed: Locations ({count})"
+    "title": "ComiXed: Localizações ({count})"
   },
   "location-details-page": {
-    "title": "ComiXed: Location - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Localização - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "story-arcs-page": {
-    "title": "ComiXed: Story Arcs ({count})"
+    "title": "ComiXed: Arcos da historias ({count})"
   },
   "story-arc-details-page": {
-    "title": "ComiXed: Story Arc - {name} ({count, plural, =1{One Comic} other{# Comics}})"
+    "title": "ComiXed: Arco da historia - {name} ({count, plural, =1{Um comic} other{# comics}})"
   },
   "duplicates-page": {
-    "title": "{pageCount} Duplicate {pageCount, plural, =1{Page} other{Pages}}",
+    "title": "{pageCount} {pageCount, plural, =1{Pagina} other{Paginas} duplicadas}",
```

Comment:
The final curly brace on this entry is out of place. It should come after Paginas as follows:
```
"title": "{pageCount} {pageCount, plural, =1{Pagina} other{Paginas}} duplicadas",
```

Reply:
Updated